package.json

```diff
@@ -7,41 +7,42 @@
   },
   "license": "MIT",
   "dependencies": {
```

Comment:
On `npm install` I get this error but everything works anyway. Looks like my Docker image misses a dependency. Not relevant, isn't it?

```
> modern-syslog@1.2.0 install /home/node/app/node_modules/modern-syslog
> node-gyp rebuild

gyp ERR! build error 
gyp ERR! stack Error: not found: make
gyp ERR! stack     at getNotFoundError (/usr/local/lib/node_modules/npm/node_modules/which/which.js:13:12)
gyp ERR! stack     at F (/usr/local/lib/node_modules/npm/node_modules/which/which.js:68:19)
gyp ERR! stack     at E (/usr/local/lib/node_modules/npm/node_modules/which/which.js:80:29)
gyp ERR! stack     at /usr/local/lib/node_modules/npm/node_modules/which/which.js:89:16
gyp ERR! stack     at /usr/local/lib/node_modules/npm/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /usr/local/lib/node_modules/npm/node_modules/isexe/mode.js:8:5
gyp ERR! stack     at FSReqCallback.oncomplete (fs.js:181:21)
gyp ERR! System Linux 4.19.104-microsoft-standard
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/node/app/node_modules/modern-syslog
gyp ERR! node -v v14.13.1
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
```

Reply:
This package also doesn't install on jQuery infra so I made it optional in the PR and I fallback to regular `console` methods. You can see it's in `optionalDependencies`.

Reply:
Ahh ok. So this should be no problem.